AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/test/TestAdminHV1Only.dfy

```diff
@@ -23,6 +24,61 @@ module {:options "/functionSyntax:4" } TestAdminHV1Only {
   import CleanupItems
   import AdminFixtures
   import TestGetKeys
+  import KeyStoreAdminErrorMessages
+
+  const testMutateForHV2FailsCaseId := "dafny-initialize-mutation-hv-2-rejection"
```

Comment:
Blocking: this test should have been removed. 
Rishav had sufficient logic written to start an HV-2 mutation. 
Will drop it, and CI on this and downstream PRs will, hopefully, turn green.